enterprise/frontend/src/metabase-enterprise/sandboxes/components/EditSandboxingModal/EditSandboxingModal.unit.spec.tsx

```diff
@@ -40,27 +50,30 @@ const TEST_CARD = createMockCard({
   },
 });
 
-const setup = ({
+const TestComponent: typeof EditSandboxingModal = props => {
```

Comment:
It's not clear why we're changing this. We should only preload data via the hooks in the test component if the component expects its to be loaded and doesn't handle loading the data himself. Is that's the case here? I've quickly went over `EditSandboxingModal` and couldn't find that it expects them to be preloaded.

If the component handles the data loading (e.g. using the entity loader / hook) it should be fine just to use `setupCollectionsEndpoints`

Reply:
`EditSandboxingModal` uses `QuestionPicker` which uses `ItemPicker` which triggers `Collections.selectors.getExpandedCollectionsById`, which I modified, which now uses `Collections.selectors.getList`, which uses `entity.selectors.getObject` which  requires `collections_list` to be present in the redux store.

Reply:
Without this rendering will crash because of data missing from redux store.

Reply:
> Is that's the case here? 

Yes, I believe so.

Reply:
@kamilmielnik if the above is true, `ItemPicker` should be fixed and it should call `Collections.loadList` or new `useCollectionListQuery` - there shouldn't be a dependency like this. Can you fix this?

Reply:
Ok, now I get it.

Turns out there was a bug and the error message was misleading. I reverted these changes.

Good catch, thanks :+1: 

7c030c1fcd3fbb5a5ee03087e4ac66caf14857e1